packages/api/src/routers/svc/webhooks.ts

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import express from 'express'
+import { readPushSubscription } from '../../datalayer/pubsub'
+import { getRepository } from '../../entity/utils'
+import { Webhook } from '../../entity/webhook'
+import axios, { Method } from 'axios'
+
+export function webhooksServiceRouter() {
+  const router = express.Router()
+
+  router.post('/trigger/:action', async (req, res) => {
```

Comment:
nice. was wondering how you would do this.

Reply:
I will create the pub/sub subscription